cli/flox/src/commands/search.rs

```diff
@@ -418,17 +419,20 @@ fn render_show(search_results: &[SearchResult], all: bool) -> Result<()> {
     Ok(())
 }
 
-/// Searches for an environment to use, and if one is found, returns the path to
-/// its manifest and optionally the path to its lockfile.
+/// Return an optional manifest and a lockfile to use for search and show.
+///
+/// This searches for an environment to use, and if one is found, it returns the
+/// path to its manifest and optionally the path to its lockfile.
+///
+/// If no environment is found, or if environment does not have a lockfile, the
+/// global lockfile is used.
+/// The global lockfile is created if it does not exist.
```

Comment:
```suggestion
/// Return an optional manifest and a lockfile to use for search and show.
///
/// This searches for an environment to use, and if one is found,
/// it returns the path to its manifest
/// and optionally the path to its lockfile.
///
/// If no environment is found, or if environment does not have a lockfile,
/// the global lockfile is used.
/// The global lockfile is created if it does not exist.
```

Reply:
style: sembr ...

Reply:
this isn't blocking (not even user facing), just a drive by formatting :)

Reply:
I didn't feel like you were blocking!